### DISCUSSION TITLE
Flame/Hot icon required

### DISCUSSION THREAD

**Discussion by ShaunHalliday** [twilio-labs/paste#3945] [Request: Icon] [answered]
### Description

Hi,

For a build in Flex we require a flame style icon, this is to denote a "hot" sales call coming in to Flex. We are upgrading from Flex 1 to 2 for one of our plugins, previously we used MUI icon for this (was titled WhatshotIcon in MUI v5) but we want to use Paste for everything in our builds now.

Kind Regards,

Shaun

### Link to icon

https://www.streamlinehq.com/icons/core-line-free/interface-essential/interface-essential/trending-content

### Icon type

- [X] UI icon
- [ ] Product icon

**Comment by serifluous** [comment, marked as answer]
Hey @ShaunHalliday ! Since this is for a Flex plugin, you can use the [SVG-to-React library](https://paste.twilio.design/core/libraries#svg-to-react) to add the icon to your plugin. You'll need the SVG, so the [Noun Project](https://thenounproject.com/search/icons/?q=flame) is a good place to start if you don't have access to Streamline SVGs, or you can grab the original SVG from MUI.
